### PR TITLE
PG new content fix

### PR DESCRIPTION
**Pull request by GijsSchenk**

### CODE REVIEW

File: Playground/src/tools/monacoManager.ts [hunk 1]
```diff
@@ -120,18 +120,35 @@ export class MonacoManager {
...
     private _setNewContent() {
         this._createEditor();
-        this._editor?.setValue(`// You have to create a function called createScene. This function must return a BABYLON.Scene object
-    // You can reference the following variables: scene, canvas
-    // You must at least define a camera
 
-    var createScene = function() {
+        if(this.globalState.language === "JS"){
+            this._editor?.setValue(`// You have to create a function called createScene. This function must return a BABYLON.Scene object
+// You can reference the following variables: scene, canvas
```
Comment (deltakosh) [inline Playground/src/tools/monacoManager.ts:126]: i guess it is engine, canvas right?

File: Playground/src/tools/monacoManager.ts [hunk 2]
```diff
@@ -120,18 +120,35 @@ export class MonacoManager {
...
+// You must at least define a camera
+
+var createScene = function() {
+    var scene = new BABYLON.Scene(engine);
+    var camera = new BABYLON.ArcRotateCamera("Camera", -Math.PI / 2, Math.PI / 2, 12, BABYLON.Vector3.Zero(), scene);
```
Comment (deltakosh) [inline Playground/src/tools/monacoManager.ts:131]: I want it to stay with a freeCamera :)

File: Playground/src/tools/monacoManager.ts [hunk 3]
```diff
@@ -120,18 +120,35 @@ export class MonacoManager {
...
+        } else {
+            this._editor?.setValue(`// You have to create a class called Playground. This class must provide a static function named CreateScene(engine, canvas) which must return a Scene object
+// You must at least define a camera inside the CreateScene function
+
+class Playground {
+    public static CreateScene(engine: BABYLON.Engine, canvas: HTMLCanvasElement): BABYLON.Scene {
         var scene = new BABYLON.Scene(engine);
-        var camera = new BABYLON.ArcRotateCamera("Camera", -Math.PI / 2, Math.PI / 2, 12, BABYLON.Vector3.Zero(), scene);
+
+        var camera = new BABYLON.FreeCamera("camera1", new BABYLON.Vector3(0, 5, -10), scene);
```
Comment (deltakosh) [inline Playground/src/tools/monacoManager.ts:144]: Agree let's move both on FreeCamera

File: Playground/src/tools/monacoManager.ts (hunk 2 above)
Comment (Popov72) [reply Playground/src/tools/monacoManager.ts:131]: What about leaving the arc rotate camera commented?

That way it's easy to use one or the other just by commenting/uncommenting.

Comment (deltakosh) [reply Playground/src/tools/monacoManager.ts:131]: Excellent idea!

Comment (GijsSchenk) [reply Playground/src/tools/monacoManager.ts]: Alright, the changes are in

**Pull request merged**